ext/standard/random.c

```diff
@@ -85,8 +89,29 @@ static int php_random_bytes(void *bytes, size_t size)
 	}
 #elif HAVE_DECL_ARC4RANDOM_BUF
 	arc4random_buf(bytes, size);
+#elif defined SYS_getrandom
+	/* Linux getrandom(2) syscall */
+	size_t read_bytes = 0;
```

Comment:
Although we mainly do this on Windows, but declarations at first but I'm not sure there is any recent (and sane) compiler that will complain here, but not a biggie.